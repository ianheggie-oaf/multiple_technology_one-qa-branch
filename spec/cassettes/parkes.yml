---
http_interactions:
- request:
    method: get
    uri: https://parkes-web.t1cloud.com/T1PRDefault/WebApps/eProperty/P1/eTrack/eTrackApplicationSearchResults.aspx?Field=S&Period=L28&f=$P1.ETR.SEARCH.SL28&r=PSC.P1.WEBGUEST
    body:
      encoding: US-ASCII
      string: ''
    headers:
      Accept-Encoding:
      - gzip,deflate,identity
      Accept:
      - "*/*"
      User-Agent:
      - Mechanize/2.7.7 Ruby/2.5.8p224 (http://github.com/sparklemotion/mechanize/)
      Accept-Charset:
      - ISO-8859-1,utf-8;q=0.7,*;q=0.7
      Accept-Language:
      - en-us,en;q=0.5
      Host:
      - parkes-web.t1cloud.com
      Connection:
      - keep-alive
      Keep-Alive:
      - '300'
  response:
    status:
      code: 200
      message: OK
    headers:
      Content-Type:
      - text/html; charset=utf-8
      Transfer-Encoding:
      - chunked
      Connection:
      - keep-alive
      Date:
      - Wed, 19 Feb 2025 03:42:31 GMT
      Set-Cookie:
      - ASP.NET_SessionId=m2gqpc45lkwh0z45ysjaam45; path=/;SameSite=None;Secure; HttpOnly
      X-Aspnet-Version:
      - 2.0.50727
      Cache-Control:
      - no-cache, no-store
      Pragma:
      - no-cache
      Expires:
      - "-1"
      Server:
      - Microsoft-IIS/10.0
      Content-Encoding:
      - gzip
      Vary:
      - accept-encoding
      X-Cache:
      - Miss from cloudfront
      Via:
      - 1.1 6a9ce19017cc8785b8da35bdfbbe4298.cloudfront.net (CloudFront)
      X-Amz-Cf-Pop:
      - MEL52-P2
      X-Amz-Cf-Id:
      - BNfu0HowH3Co4ctOJORA_ia_KgPrZaciJBJaC6wb8Y_nv-6-CthPoQ==
    body:
      encoding: ASCII-8BIT
      string: !binary |-
        H4sIAAAAAAAE/91cfXPiONL/e7dqv4OOZ25npnaDgSQzyQzJlY0NIQEC5i1wdZUStmI7yC9rySHkqfvu15JtAgmZJLe7M3ObGWPrrdVq/brVLRl++vGnH6t/089rg0nXQC73KeoOtVazhgo7ijLerSmKPtDRxcmg3ULlYgn1eexZXFGMTgEVXM6jT4qyWCyKi91iGDvKwFRuBZWyaJY97jDZpmhzu3AsOxQ10K1PA3a0hUb58PAwbSqqV12C7eMq9zgl0PqHLo7nhKG+68UE1cIksDyKdlA/mfke58RGLcw4qhwgHS8ZDE5JW1apF8yRG5Oro0KxqMB/NYouBy7xCVO6/ZqikyucUF60GCsgvozIUYGTW67IdEzoUYHxJSXMJYQXkHJcVVzBGPQwC+0lcPbTjwj+qsyKvYivU7jGNzjNLSAWW3n/fVmRKde/JSRe7uwWPxTLRd8LitesANTTFsdICAzooupVGPsowD4whlkUEF6HjALyCXdD+6gQhQz4whb3wuCoQAYxtuYwROpZWGT1CY4t1yQMxsiKmEW3/6h7hNpH/Z+xH33uktgDKq3KgUxeHf29stctF42BWewbqlk7KfbzsvgIxFWEwrGhNYZGf1BAHvQPJHOeQBhV27sRNy+IklwYrmfbJChkY7i8HDWNcX+gDoyUwEbGDaYJOSooC6OrL4Zn7YGx39adu7Y+L3UWh/O6c+rrld5YdZqauGtq08juMq36k+5UmxiqY4h7bepkbRYHwPWeMWqM9mx1eGqOzciuaZ1ZxaQnDtdnlTKfjPfnA7/Op70h1Kvf4du9bq9yyGfjejI16svp7ohPx/slzYx67eGoN5x3ytaueTMb07vJhRnNKvv1aYOWBuPD+dTcO+2NO9fTi85dvzLah3SvV6mXrcrobuTXGR6brt2gNzPfIbW66U7mNJk2RntqT1VPHHs4qxyyUYMupuZed6P+3F7OdkeL6dicgwy2jPFA1Xrl/nQ8XULfYdPouJOKS08WRl2DfqwKu1KdIRnq9fqwfqrbF53SrFKOrEq9hMeHydAf+ZY/usOrdh1oa4Yni6GmquqV5g+vtMpoPlWbug4yVd29k5FP7/D4lE3HLsxPs6Y6B92petCt15iqmpPGiWOeT8ZlqvUiZpWm5ZnfkX2dOLcG8Ona49uSGKM9HtaNpdlr9/fro3ozGZZG2nDeabZop92+dohONdcS83NhUotO3dnJiE61Xqz2jJpaMeeqYwl5NMTc67eT2slCo9NGr6ZK3MDdAZ6c5pWqDtvDyghodUA+Yh5Gd2m9Vfmkfd2+aevGTftaXXb6Wmm2VG/P+weLtnewbOvNsqiv9UT9HqkZ9hLmc0guNBjjLfB+qvVKfGAO6/1eySFGWq5bPi1NL06jyfA2snZ7y4+9nqrCf7WrwN+BkK+a/i1q4j6s925bVIXLphNvvzvxIzrZNbVpo76wG6OltVw4Ix/uFXozuy7dtmt7pda1mrRrC6e3O2L2yWg5HZST6cWoBHJjrZrWs8enDI87rekFHc4qnM6uy4k9vmWaakCXNUsddfYB03TW39dnlVs2HXfkXFnL/RPAPoUxXLfmgn/TaY/4bKQbyRCuUWXkteaHHvZH17ZRn08uNDHWu9ZcyuZ+7H3gGep/sU1NG4OcAFOHSXdgLFrXPRjT3gL414HXkn1xSrvjfWpfmMvJeMGada088W+jSQlWipF5gyujBMrLs8atfdFrw7jUxkUFZD8u32mjwzt8EdFa/bCS9yEEf1p+Ttb7JzD2eW/3NBKyx+P67KLXFLSbNXWuiwdVzKUq5KieqAtV/GmrD7VHXy2rlP6pJKKWdK3XE+T+CzpD0e5kdtEZT8a35WnP5VZ56gr9aTv73CodLkAWrt04XIK9m097EeR1bqCcz+Z0rvvlu+HJ6Y3VGC3tE3o+GZep2jM0uGoq/GmOpD/e1Bn1tn0dLto6XDVNG/QmDaiq1q+N/dadumzdNRft66HTHhgfOoO5uJyhUWpoUm6nRs+ZSOHqw9ubacOms/EosQ2hq8NT2anK5/VsLdDUpqZXemNVrAnSJh2oWo/K+WrWzX3ge81eHTRx40Fd8/Z8ttSimT+9sfwyyAH0ydMia6m5tl+PZo26N2sME9D5iboY6aCDrK2r817lMAEsJHbNvAY72gd7W56dmHcmYGRaM5fTC+1iOj6VfJw2OqWZri5OG/ZyWgd9nku9GE7Gp2WgvwQbSu3FwcHZQj1Te05NHffqvbyNAXMRmHQW9OaT3dEd2Og7oQdWSdDozcFeL4T+Zf1e243Fol0z5/bJ9CLvf9Cgd3Z9Gk0vLGmXNbCX04YJMlRreqNXP9nS16xxyoaVkbSTZqMO+tzu4obJo8Cc7xuno11KvIskPLmrlRPtdlIZLI6OhIMEnpFiSy8AnvKHH6ov8ggaRscw1cG5+cg1WCvJfARDPfhQ+1D5mPb55Q6MkdEZjNRWU1cHzfNOTv1RdkYa3I9xc9FyPv5yU6qpZ7v7hydn+qK1e/BLv6T1WnppqiiQr19HdwTutd8m45bunP1SjhrXutNKYh7s19QWS04/RqrTmoeTcR/anSbnVwc150wPQ5kOszSOmjKtJIkob2lp+ZmfEJl20nSrnqWVLD1P02eNjH7MfxH1z3iwJ+id9bmkd0bC/aCjqWd6cpPs1pxWPZzK9G9JV6bnabp1mqbP9DAS5a0wSyvOwbhd651d+6dXZi2Y4DPHn3ttah0EN7OID8yT23PjKh756VRUs+kX3qu4qgAC6SrSENte4BSORe7alXrQIuPJphZjWpgENhOhwX192SYnL+rgICBxAYELKooeXtW/7eygVuh4AdrZeaqOIGdRzNhRgUXE8jClXjAX/T4kl6c3+clz83uVRThAGUXZ+ZdorVphKTKL01LpEj5lw0vLt+VDAWHLIozNyfKo0IK4RURJRwVVZqIzsvyEINeNydVR4RrnYciny0s77IaMa9iav3sLVEulN/ApSb7Jab/99e3b94VjmVlV8MPJyhlc3b88elRVxPifEHZO5AkaD+GQVxd3OZdGAHFfPp8iN78eE1yhcAUTGGH41FRUcRYzupxHDGLdSIafOwsyK/KyRcPELlqhrwzKXVMnVzihXBmTGQReTCHdOIxIzJdKt6w0EsL4SeiTNPZ6FET9fJWHXF2z2Bga/cFJ2yiOmuPURsHsiPn3bUGig2+e4leMu+r5DmLxKtAEZi4Hrgx0cxabPnYIUww/4sui412td+KJsrwbNAtjm8RHhVIhh64LgxAcQAz8WLqyfwDL45Jtc7g1b03vLEpwLIJhyHuAnPVprIUBJwFXY4K3yWW96gCzeYQDUkDpcNARKgAQOmE39gK+rbUY0ePRiFxB9wFXIju/tjfKS8VdEEjZOCr4JEi6OCD0KR5EA3G9nuwJwTCFz9EVtMWVminPPiqkmMsldhlHcRuYbHmMC0NUKl/SGRU5jThMoryT8wCsJEFdvIQRcfYirZfdPlgsRN62q5rQdZHJvgVLYng/VKl3vNJY2OIhwIYC8KeehbkXBpAUXBUxi27/ER91+7X1fYzNnY/upKh2u/2i0ekVjtUoymnkQxMmsaqIDsWFH5gJsSWWmopiwBZFJ7wp4gRq38C6t+OSmCjLMIl3otxCxJiDPrrhYoeHOxFeKoVjncx4GH+5u3yMpmj+ytGBf2Vkw5PNX9bTGHqKX9nTGPZ4zFSQsvmjnsBBVBL6BV3KkVCdxU+anbzOd6Mgla0Ksg6lQYytOSDiG2oJkTysMdUHk2u5L1KRwWpz8JGSpFRWOvLXnODdrRPcTWZgK1An5J5FvrEBTHnJWFHWU+kEvWia+1qxO9Q654NUhVMq2fiQEfyWePFyNdMPTXBaO+dApXQj43X9q62UA6CCUjIZFyBlDFvmYIj/mkDbu9y21A4ZiVEzuApjH4vVrfoSD1tYyW1ul8h/eL1qrRXcKF0IlxbgMZqEEf6iye2axWHfMLtqvy+93eN6GDshB38OVgnGBLEnsSW7NIkDHgmJX9qbaTREh8V2Z1A4zhuvuvja+HmB1/P0era95CWz+7jlU6221ARnPI1JGRvg1J02CQW/wG5BgMxy1/p3eNYCiNI5vndDTc9xuehOOMmXnEWbXVppoM6BH1FBetGSwlpB7qJmLZHkNh83YB38jw3r9cTSCNiGKO/pM7TBWDqSz7iXg9Xa2R+MC8f9ZOZ7gHobDVyPoTEh83VMgmVb+bX/JVut17LV2mCrhRn/M9gatF8rrfYjabUh/nP/aHG9lq/WBl9SXH8KXx9fKa/Wx3V5Sb4+Ih0v0zUTQpI/CPat8t5rOSvvPWKtvPen8JYda79cJ1uVg0e8Ze8U/H6g6VJQ2ZH7a82FLsyFTjiJfS8gf6i92ODrtfZCb4G9WONLIu2PsWMbfL3WYOgDUMw1vqR9/YM0c4Ox1isthi4sxhpjUmB/CmOvNBl6C0zGQ8b+KJuxKbLX2gwdrIzYGVmhXwrtDzMam8y91mjo0mg8FNwWq5HudTx0T+phCMMqHP8czFj0eeWXZB7iYz9MOEni2l7ymlzJiXS1GDshNFIDW7XBuog9NdHDtiuln3+m94f18jG8NP9hvcdU1zmtwapPtu/ZymqZ52cxltUch/EcziAs8tS4wMXyxL7nqsUgjMTuM4Yl8iFvIv2YP5ErrqpbTl1keWyTOp6XbrkLu+4Dj1Pg4N7hkxBeocQtv6qz7SxszxWMpdf28jQ3/ayCoIK1IWQivLQSZmzfMcrc4gJifEnJUcH2WETx8pMnN4R3ZjS05p8Xns3dT+VS6e+fhWBFJ9lw13f2vswCndF080LHnEC0kc2z7ACICppbGIdWdY9Qu4N90i0bA3MbjStRo4VnYhc+LQdvBDrJIuunhJLTHomX1l5EvLKrlMpKpVTZRzxE5UOlVJGpvKP1SD6Vxk8/pv8gQgnm6NH7iHwZgcw5ueWKxUAorjzye+ZYCnb98oOhmEDAvgNNi3AVFJgUiIRXgZ5E8RoEsrluxJ59GQW0ljAe+h7DYhdCZAoF+yHnW0Bu/VmkxSWVVOrbl8jHJILzNEH00pL1oL8+sURPQq1g81R2lvUm6ObXU7MFEF7n/1EHVOBrowOUQYw96BfkR730ZAFlcMkoZ/OYDjtnaNt9Hfbr5fIM7/XC8XwnE06fRBgOFUI4BN/kfq2AbT0jhFMR5fH5IKb8qLBS7vRUcCdV51J0+9klHsTnn8RjmrkvVPzRXv1PP/6wWhQ2hSPQvSEAAB+g8IcfqhzPKMmnwAHIwUkkoRT03ILZT08mIR1h25bp3cL6oeXrZejEdoa4AZ4lFMcjjyxWI09HJy1YJgQrpBRHjHzKH8C0wTgF53HOtkswHMGZ4SIFqyhzEbPCCJQWmoHZwtkB0jOn9JkSvtkE8Zt7EL+RWvJm6yDgRL8fxvyNif2mLc721wEsdkPS0Ia7x1X+DdlrhbZDfBJwYeAFm6sMaY2/DyZ1wqzYi4QdEiyuJb8P/roxnOPHS3k+usac4FXmoTRTDuC74riGOXHCeLnGn2A6z/7++IYNax9zTsBntmEZZYLbVR5SbZn5fYg40/eACy9IMrrKQTJrxWZqwBQegylYWbFAnDTQDStm35+3/5mWS1q1SvoJVkyIuFYD92lPKZX29zOugRduH1fAmUpdq6oCySpccI7fatbkG3/INAZDs2PoaAedJ3yWeFSsGpDqu8RetQAzy3icSC8A1UjMvSux0IvjPKAnaD5VA+VfSWkylqxRhNjCjV/wGhbQ5WD63lgJMwlLKGfC93mhgX9jgYuUimk3E9OHQzhHcQjq85gQjrqqeWb0Uac/RpWDj6UNyaFaEsf4Ogwc1KU4CDzxkL2ggOSWFerG4TWxOGrjAEIZHxiVItuKFkw5iQPQ5m8GmN1UHpkkJGD2ATCl3Y1hbwHMXxQZqTxyZHz8gBohteOEucgMsf1FbAzicIn0MHEoZmjsUephH7XhTuJfkYkJUgOHUO+3hGS5T+PiG1uRvQ1Q6CpYEQGKcnkDFOXd+5As03jVvhGGgAm16HtOAAogBymsgU5uCA0joRAoM6ncC4P7chWdcxeOa991Qo5YRCxpUChdIogfxHrnMWK/X9X/etYiFUeOifI+6mNKlji22fOgQEIMqSmIJevfqR3Y3z7lpYONKS99fDTlNRcDrFF4hYaMwBrRDOwwjJFJrJhgMcGoji2Penwph/8SJNRC308CaJE39dYWla837alI8mmv7L1m2lVgHyONAEh+RUgFWaBBuAhWi4aUxVYofGPV//AEDj5s4uCx6hsxLHqgzwIJGHXIAsFyCt5BEgtUbHgOz1kCVayKWGCHIRzYwj/00hQs954NK6qHqZSgQNPXA0QqmxUgdpEWLmyPWy6qxRDqAF/oS75DG3Owb4vV0tCPiAzH5Ui2YuHbuwcfn4DDx98HhxqOI4iy5ez+JaCRymkFjT00JoyHgdCB51zK0xD8SALGJbBjYTIYDvjTkPjG5uHgCTwcbuChXHm0TPwPm4dsp0uOvLR910iWpbLJMVCuQJgTznHkwdw+H1i0iERB27NcQilB3SSw3Kdh8O0tw6EccyULHHQ19xE346VnkGASjj0ZRI0xvTfp/6sLRCqTHAFwXmVQ7w7PCOwUwmL4nCHoYO7iwCMUdV1wmCKke7eZf7x1dfi2pqAsvw5VegSA0oOthgMlO8WRYBYLtr4AhIsIYUfEyMkVtjg4CTY6CX1yX+mZaGHNEwAXO3BAZXK6KxpfzTPIZJFP/H7lFVGjRoJrDAfJ6DR0A5AB7OvKAWyd8m+u9uXyhto3z/vtTPFLD4JD2GKqZOd1WXDYDBgHLUfnwU7f42AVyQLiQ4EBuPnCe+ovGSe+HL4ASiu0MAVRQkwZyPK1sJGtauVkw4AJqgsMNNA7RhYkBrrvkb/ah0Fsk/7Xw0cqtRwf5b1XLQ1IGwAu/CwG2oqLb2wKKhugaDVySDzwEj888grAD4yXKEw46vMw9hewExYjPYY1AaYOiZcRVtP8MjDcEwT3KycIfpUXCIKLMJ6vCH692U/Fk89+38U09NHuxwoS3y1FE+z7GD27u6TjG89GaszdJM5DiFQJtiLi21uKdCctXx/uQbG5s1j6EigkHn5FUpdlrPDspL4WJbKHrIN0+/bZLr4eblIJ5rjZraAaxbEX2Bi0JSYv36nWPYZn4FHwJeqT+MazCEMtzwdraUttuAeQeIIsPKPyd6RW59Brp86bR9LbzuY3azyVEofZOX3xnF6io7RF1fZukEUxY0cFbInwX0s4hFSsRa64OCIWVcW18WsJTL4jXkAB9sWxMWz1l97UwoDD6vJmxoNuTG68MGEFdINpAjXuMzzxVpOof5nVv9yov/Yd8m4Byd/L2vwOOeRm3AoG1TgOFynD6Rf9Jaer99TSIYq89Wt9yBYlOIYz76zJerX8eZvsf09ZPhs5jfy+LX9rXnRcVaLjzbFVo3wSveDGYx4gS4zqUTUx23IKGNNCmGdfg+HnTS3GvvDtYym2h21bxMH00a8nILTJHcr+QKclATn/bpS9rVMPQ07ilhfMywW0BoDRVgBArhuL0yuX8+hT+otvERY/xFYM7r/eGsXeDbaWQAHHDuFHhUtG6FXhuJvmizD6CRY931l7ZwtSKXt9Ap1gWD2BxwyA4C9xz8JUipA99wLLiMR5bY1ia55+1x1vvMviEk+8ulK+f3elHN1+noWxTeK1N1zuoZ7JdXV7TsCVDQEPQDziFcBNDYPcFwjY9hiIwfNJDEQ2ZQx2MCt6lZglClZiBk6BCAOr9D8o5t0NMQ9BQo/FDLkvELMVJgFs6ysWGFcwzztZGihuylwYUygHh1fWF4JfgWLz4QnYswgHa7hvh7CSEaGT62vAJiWEHrwaJl/V+q71QwzgoY6kQx3JN6vSGbnGX/x5EjBc922yjZk0Awkqr5e9IoT/wKBLTp9YlbZa4loYLWNhPgrHP1vw/BkNiOUGIQ2dJcSCBGWuCBJRw9Z1ZUtv92BZbyF8iGr6zhFae9vzXmwCM4pS/ds/a7o6UP/5049v3tmhlfgk4O+LMcH28t0V4FR4G+jde/T/3hV69+bd2/+z5MIAPoMRx2HM3r4veuzd20/Zgvb2PTo6QjxOCDQR1S8v+zXzvNXqnvebg+Z55wIaQON3b0tv33/eVmGyXuHf/37/WVH+9S8QfFVJRyMfryBCkg+z0F7KB5f79PinH/8Db/mFTKFUAAA=
  recorded_at: Tue, 14 May 2019 14:00:00 GMT
recorded_with: VCR 6.0.0
