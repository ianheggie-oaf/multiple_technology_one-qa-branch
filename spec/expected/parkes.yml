---
- council_reference: CC2024/0055
  address: 69 Page Street PARKES NSW 2870
  description: APPLICATION RETURNED - Outbuilding - Shed
  info_url: https://parkes-web.t1cloud.com/T1PRDefault/WebApps/eProperty/P1/eTrack/eTrackApplicationDetails.aspx?ApplicationId=CC2024%2F0055&f=%24P1.ETR.APPDET.VIW&r=PSC.P1.WEBGUEST
  date_scraped: '2019-05-15'
  date_received: '2025-01-29'
- council_reference: CC2025/0003
  address: 76 Goldrush Road PARKES NSW 2870
  description: Outbuilding - Shed
  info_url: https://parkes-web.t1cloud.com/T1PRDefault/WebApps/eProperty/P1/eTrack/eTrackApplicationDetails.aspx?ApplicationId=CC2025%2F0003&f=%24P1.ETR.APPDET.VIW&r=PSC.P1.WEBGUEST
  date_scraped: '2019-05-15'
  date_received: '2025-01-29'
- council_reference: DA2025/0005
  address: 52 Goldrush Road PARKES NSW 2870
  description: Dwelling - Manufactured Home
  info_url: https://parkes-web.t1cloud.com/T1PRDefault/WebApps/eProperty/P1/eTrack/eTrackApplicationDetails.aspx?ApplicationId=DA2025%2F0005&f=%24P1.ETR.APPDET.VIW&r=PSC.P1.WEBGUEST
  date_scraped: '2019-05-15'
  date_received: '2025-01-28'
- council_reference: DA2025/0006
  address: 23 Bowditch Crescent PARKES NSW 2870
  description: Erection of a New Structure - Shed
  info_url: https://parkes-web.t1cloud.com/T1PRDefault/WebApps/eProperty/P1/eTrack/eTrackApplicationDetails.aspx?ApplicationId=DA2025%2F0006&f=%24P1.ETR.APPDET.VIW&r=PSC.P1.WEBGUEST
  date_scraped: '2019-05-15'
  date_received: '2025-02-03'
- council_reference: DA2025/0007
  address: 24 Weston Street PARKES NSW 2870
  description: Erection of a New Structure - Carport and Shed
  info_url: https://parkes-web.t1cloud.com/T1PRDefault/WebApps/eProperty/P1/eTrack/eTrackApplicationDetails.aspx?ApplicationId=DA2025%2F0007&f=%24P1.ETR.APPDET.VIW&r=PSC.P1.WEBGUEST
  date_scraped: '2019-05-15'
  date_received: '2025-02-03'
- council_reference: DA2025/0008
  address: 24 Saleyards Road PARKES NSW 2870
  description: Change of Use - Indoor Recreation Facility
  info_url: https://parkes-web.t1cloud.com/T1PRDefault/WebApps/eProperty/P1/eTrack/eTrackApplicationDetails.aspx?ApplicationId=DA2025%2F0008&f=%24P1.ETR.APPDET.VIW&r=PSC.P1.WEBGUEST
  date_scraped: '2019-05-15'
  date_received: '2025-02-07'
- council_reference: DA2025/0009
  address: 12 Cookapie Street PARKES NSW 2870
  description: Erection of a New Structure - Shed
  info_url: https://parkes-web.t1cloud.com/T1PRDefault/WebApps/eProperty/P1/eTrack/eTrackApplicationDetails.aspx?ApplicationId=DA2025%2F0009&f=%24P1.ETR.APPDET.VIW&r=PSC.P1.WEBGUEST
  date_scraped: '2019-05-15'
  date_received: '2025-02-12'
- council_reference: DA2025/0010
  address: 28 Elizabeth Street PARKES NSW 2870
  description: Erection of Retaining Wall
  info_url: https://parkes-web.t1cloud.com/T1PRDefault/WebApps/eProperty/P1/eTrack/eTrackApplicationDetails.aspx?ApplicationId=DA2025%2F0010&f=%24P1.ETR.APPDET.VIW&r=PSC.P1.WEBGUEST
  date_scraped: '2019-05-15'
  date_received: '2025-02-12'
- council_reference: DA2025/0011
  address: 15 Saleyards Road PARKES NSW 2870
  description: Advertising Signage
  info_url: https://parkes-web.t1cloud.com/T1PRDefault/WebApps/eProperty/P1/eTrack/eTrackApplicationDetails.aspx?ApplicationId=DA2025%2F0011&f=%24P1.ETR.APPDET.VIW&r=PSC.P1.WEBGUEST
  date_scraped: '2019-05-15'
  date_received: '2025-02-13'
- council_reference: IOSM2025/0001
  address: 14 Cookapie Street PARKES NSW 2870
  description: Install On-Site Sewage Management System
  info_url: https://parkes-web.t1cloud.com/T1PRDefault/WebApps/eProperty/P1/eTrack/eTrackApplicationDetails.aspx?ApplicationId=IOSM2025%2F0001&f=%24P1.ETR.APPDET.VIW&r=PSC.P1.WEBGUEST
  date_scraped: '2019-05-15'
  date_received: '2025-02-19'
- council_reference: LG2025/0003
  address: 32 Clarinda Street PARKES NSW 2870
  description: Carry out water, sewer and stormwater drainage work
  info_url: https://parkes-web.t1cloud.com/T1PRDefault/WebApps/eProperty/P1/eTrack/eTrackApplicationDetails.aspx?ApplicationId=LG2025%2F0003&f=%24P1.ETR.APPDET.VIW&r=PSC.P1.WEBGUEST
  date_scraped: '2019-05-15'
  date_received: '2025-02-06'
- council_reference: LG2025/0007
  address: Shalom 372 Back Yamma Road PARKES NSW 2870
  description: Carry out Stormwater Drainage Work
  info_url: https://parkes-web.t1cloud.com/T1PRDefault/WebApps/eProperty/P1/eTrack/eTrackApplicationDetails.aspx?ApplicationId=LG2025%2F0007&f=%24P1.ETR.APPDET.VIW&r=PSC.P1.WEBGUEST
  date_scraped: '2019-05-15'
  date_received: '2025-02-06'
